keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2025,6 +2006,132 @@ Beta:
   - Automatically manages versions of mandatory/suggested configuration
   - Discrimates paginated LIST requests
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
```

Comment:
It is also necessary to enable the API group introduced by this feature.  That means adding `--runtime-config flowcontrol.apiserver.k8s.io/v1alpha1=true` to the kube-apiserver command line.

Reply:
So after reading the source, I believe all three of these must be turned on, right?

1. set `APIPriorityAndFairness` feature gate (default: false)
2. set `--enable-priority-and-fairness` cmdline flag (default: true)
3. set `--runtime-config flowcontrol.apiserver.k8s.io/v1alpha1=true` cmdline flag (default: false)

Reply:
right